config/locales/en.yml

```diff
@@ -547,6 +547,8 @@ en:
       informative_text: "Your given mortgage, deposit and grant total is £%{mortgage_deposit_and_grant_total}"
     care_home_charges:
       title_text: "Care home charges should be provided if this is a care home accommodation"
+    buyer1_not_livein:
+      title_text: "You told us that buyer 1 will not live in the property. For %{ownership_scheme} types, the buyer usually lives in the property."
```

Comment:
It seemed to me above that there can be interruption screens for buyer 2 as well? If that's so surely this message needs to have `".... buyer {buyer_num} will not ..."`

Reply:
You're absolutely right!